operator/src/main/java/oracle/kubernetes/utils/OperatorUtils.java

```diff
@@ -176,4 +176,14 @@ private static StringBuilder toStringBuilder(Reader from) throws IOException {
     return sb;
   }
 
+  /**
+   * Returns a new String composed of copies of the CharSequence elements joined together with a copy
```

Comment:
The comment says CharSequence, but the method takes String... And there is no need to mention "copy" as that is how string concatenation works.